shared/translations/languages/de.json

```
@@ -1014,11 +1018,12 @@
     "By continuing, the required amount of DFI will be converted": "Wenn du fortfährst, wird die erforderliche Summe an DFI umgewandelt",
     "I HAVE {{totalAmount}} {{token}}": "ICH HABE {{totalAmount}} {{token}}",
     "I WANT {{token}}": "ICH MÖCHTE {{token}}",
+    "incl. stabilization fee ({{dexStabilizationFee}}%": "incl. stabilization fee ({{dexStabilizationFee}}%",
```

```suggestion
    "incl. stabilization fee ({{dexStabilizationFee}}%)": "incl. stabilization fee ({{dexStabilizationFee}}%)",
```
can check all similar translations ^

---

updated!